apps/src/aichat/views/model-customization-workspace.module.scss

```diff
@@ -95,3 +96,30 @@
   padding: 5px 16px;
   margin: 10px 0;
 }
+
+.fieldLabel {
+  margin: 0;
+}
+
+@keyframes fade-in {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+.tooltip {
+  animation: fade-in 0.3s;
```

Comment:
As discussed via [Slack](https://codedotorg.slack.com/archives/C06FELVTV0Q/p1720624390509899?thread_ts=1720476078.818049&cid=C06FELVTV0Q)
could we left-align the text in the tooltips since the descriptions are a bit lengthy?

Reply:
Done!